pyproject.toml

```diff
@@ -18,7 +18,7 @@ dependencies = [
     # Not directly used on the client, but some server-side environments have
     # tblib transitively enabled, so this is needed for unpickling.
     "tblib~=1.7",
-    "tiledb>=0.15.2",
+    "tiledb~=0.25,!=0.33.0,!=0.33.1,!=0.33.2",
```

Comment:
I'm not entirely sure what the lower band should be 🤔 

0.15.2 is too low, it doesn't immediately install on a Mac. Our CI tests pass with 0.25 on Linux, so that seems like a good candidate?

Reply:
Nevermind, we require 0.30, the first version to provide `ls_recursive()`.